precompiles/warden/IWarden.sol

```diff
@@ -113,6 +113,12 @@ enum BroadcastType {
     Automatic
 }
 
+enum OptionalBroadcastType {
```

Comment:
Tbh, if we just made BroadcastType to have Unspecified value initially, this problem wouldn't occur

Reply:
Agree, but it will require migration for existing sign requests

Reply:
@backsapc I did so initially, but as @Pitasi pointed out back then, only 2 enum values are better in this case, since this way we didn't need any migrations